firefox-cliqz_privacy_notice/en-US.md

```diff
@@ -1,137 +1,176 @@
 
-* **Snippets**
+* __Webpage data for Snippets__: When you choose to click on a Snippet link, we may receive data about the link you followed. This information is not associated with any other information about you. [Learn more](https://abouthome-snippets-service.readthedocs.io/en/latest/data_collection.html).
 {: #snippets }
 
-	Firefox's default home page (&lt;about:home&gt;) loads small bits of information right below the search bar that we think will be useful to you. We call these "snippets". About once per day, Firefox connects to Mozilla and provides you with new snippets, if available. Mozilla may collect how often snippets are clicked, snippet name, browser locale, and which version of Firefox you're using. We only retain this information after 60 days in aggregate form.
+### Use Cliqz Results {: #cliqz-features }
+Firefox with Cliqz by default connects to Cliqz to display search results in the URL bar. Cliqz GmbH is a German company offering a privacy oriented search experience.  Learn more at their [privacy policy](https://cliqz.com/en/privacy-browser).  When the Cliqz feature is enabled:
 
-	To help display relevant snippets, Firefox sends Mozilla a monthly request to look up your location at a country level using your IP address. We then send that country level information back to Firefox, where it's stored locally.	Firefox will then choose snippets to show you based on the locally stored country information.
+* __Search Queries and Webpage Data to Cliqz__: This includes text as you type in the URL bar, queries you send to certain search engines, and data about the webpages you visit and interactions with those pages, such as mouse movements, scrolls, and time spent.
 
-	Some Mozilla sponsored snippets are interactive and allow you to optionally share your phone number or email address. For example, you can enter your phone number to receive an SMS to install Firefox on Android. Your information is received and handled by our email and mobile marketing vendor.
+* __Interaction Data to Cliqz__: This includes your interactions with specific fields and buttons in the Cliqz feature. This data is tied to a unique identifier allowing Cliqz to understand performance over time.
 
-* **Firefox Health Report**
-{: #health-report .inproduct-link }
+* __Interaction Data to Mozilla__: This includes counts of visits to search engine pages, which search engines you use, and your Cliqz identifier.  
+	
+[Learn more](https://cliqz.com/en/whycliqz/transparency) about Cliqz data collection. To check if the Cliqz feature is enabled or to disable data collection:
+  * type "about:addons" into your Firefox URL bar
+  * select "Extensions" on the left menu bar
+  * if installed, Cliqz will appear on the right
+  * selecting "Disable" will turn off the Cliqz feature and stop related data collection
+  * selecting "Remove" will uninstall the Cliqz feature from Firefox
```

Comment:
I will need to test this locally to be 100% sure but, my first instinct is that you probably want an empty line before the first bullet, and do not indent them.